### PR TITLE
add Captionator v0.5 with git autoupdate config

### LINKED ISSUE #6176: [Request] Add captionator.js
**Library name:** captionator.js
**Git repository url:** https://github.com/cgiffard/Captionator
**npm package url(optional):** 
**License(s):** Copyright (c) 2012, Christopher Giffard All rights reserved.
**Official homepage:** [Captionator.js](http://captionatorjs.com/)
**Wanna say something? Leave message here:**

I wanna be able to use .srt subtitles without fussing about converting to .vtt or whatever. Being able to grab this lib from CDNJS would be cool.

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by pvnr0082t**
PR for #6176 
@x09326 Please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/cgiffard/Captionator
 * Official website (optional, not the repository): http://captionatorjs.com/
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 205
   - Count of watchers: 14
   - Count of forks: 49
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #6176, cc @cgiffard

### CODE REVIEW

**Comment by x09326** [comment]
@PeterDaveHello I think the "cgiffard" in the keywords which is name of author.
Does keywords need to include it?
I just find this problem. Please help me check it, thanks.

**Comment by PeterDaveHello** [comment]
@x09326 Feel free to remove it. Nice catch.

**Comment by pvnr0082t** [comment]
@x09326 I updated the keyword in ```package.json```. Please help me review again, thank you.

File: ajax/libs/Captionator/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "captionator.js",
+    "js"
+  ],
+  "license": "BSD-2-Clause",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/cgiffard/Captionator.git",
+    "basePath": "js",
+    "files": [
+      "captionator*!(-example)"
```
Comment (x09326) [inline ajax/libs/Captionator/package.json]: @PeterDaveHello I review it again. I don't know whether line 26 is okay. 
But @pvnr0082t said he use autoupdate then it worked.
Please check this pull request again, thank you.

Comment (PeterDaveHello) [inline ajax/libs/Captionator/package.json]: The syntax is fine, you can take a look at this reference: https://github.com/isaacs/node-glob#glob-primer

**Comment by PeterDaveHello** [comment]
Don't you need to add `css/captions.css` ?

**Comment by pvnr0082t** [comment]
@PeterDaveHello https://github.com/cgiffard/Captionator/issues/48 said that ```css/captions.css``` don't need to be added. Is that I missunderstand?

**Comment by cgiffard** [comment]
It does not need to be added. It's for the example.

**Comment by PeterDaveHello** [comment]
@cgiffard Thanks.

**Comment by PeterDaveHello** [comment]
@pvnr0082t would you please leave `basePath` blank? It'll be better to not use `js` or `css` as base path.

**Comment by pvnr0082t** [comment]
@x09326 I updated autoupdate config in ```package.json``` file. Please help me review again, thank you.

**Comment by PeterDaveHello** [comment]
@pvnr0082t but you didn't change the files in your commit.

**Comment by pvnr0082t** [comment]
@x09326 I have changed the files. Please check it again, thank you.

**Comment by x09326** [comment]
@PeterDaveHello I think this PR is okay, please check it again. thank you.

File: ajax/libs/Captionator/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/cgiffard/Captionator.git"
+  },
+  "keywords": [
+    "Captionator",
+    "captionator.js",
+    "js"
+  ],
+  "license": "BSD-2-Clause",
```
Comment (PeterDaveHello) [inline ajax/libs/Captionator/package.json]: How do I know it's BSD-2-Clause? I didn't see any related keywords in the project, if you use the some license detector to match its license in readme with all the well know licenses, I'll respectfully ask you to confirm that the author wouldn't mind the project to be marked as "BSD-2-Clause"

File: ajax/libs/Captionator/0.5/js/captionator-min.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+
+/*
+	Captionator 0.6 [CaptionPlanet]
```
Comment (PeterDaveHello) [inline ajax/libs/Captionator/0.5/js/captionator-min.js]: What's the reason why it's 0.6 but not 0.5 here? Don't you get it from this url?
https://github.com/cgiffard/Captionator/blob/v0.5/js/captionator-min.js

**Comment by PeterDaveHello** [comment]
@x09326  We don't use **check**, try to use **review** as the keyword instead of **check**.

**Comment by x09326** [comment]
Got it.

**Comment by PeterDaveHello** [comment]
@x09326  I noticed that your review didn't really check the detail in the commit, that's a bad signal, you should take a look all my reviews after yours, that'll also help you send good PR/commit.

File: ajax/libs/Captionator/package.json (hunk 2 above)
Comment (pvnr0082t) [inline ajax/libs/Captionator/package.json]: I confirmed with author before.
https://github.com/cgiffard/Captionator/issues/49

File: ajax/libs/Captionator/0.5/js/captionator-min.js (hunk 3 above)
Comment (pvnr0082t) [inline ajax/libs/Captionator/0.5/js/captionator-min.js]: I forgot to checkout to v0.5. I fixed it.

**Comment by x09326** [comment]
Sorry, because I don't know that are mistakes.
I will learn it from your review, and try to find more odd parts.

**Comment by pvnr0082t** [comment]
@x09326 I have fixed some problems. Please help me review this PR again, thank you.

File: ajax/libs/Captionator/package.json (hunk 2 above)
Comment (PeterDaveHello) [inline ajax/libs/Captionator/package.json]: Okay, next time you should add this info in the first commit, so that the review can be faster and efficient.

**Comment by PeterDaveHello** [comment]
Okay, let's move now. BTW, @pvnr0082t you should open an issue in the project repo see if the author forgot to add a tag for v0.6 as the files you added wrong in this PR show they are v0.6

**Pull request merged**